Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -359,6 +362,10 @@ class Connection : public ThreadSafeRefCounted<Connection, WTF::DestructionThrea
 
     Timeout timeoutRespectingIgnoreTimeoutsForTesting(Timeout) const;
 
+    virtual bool isMainRunLoopConnection() { return true; }
```

Comment:
I don't think these need virtual functions
(and the solution probably should be something different than BackgroundConnection anyway)